tests/ci/bumper_service_test.ts

```diff
@@ -0,0 +1,364 @@
+import { Rhum } from "https://raw.githubusercontent.com/drashland/rhum/rhum-cli/mod.ts";
+import { BumperService } from "../../ci/bumper_service.ts";
+
+const b = new BumperService("rhum", []);
+const c = new BumperService("rhum", ["--version=v1.2.3"]);
+
+Rhum.testPlan(async () => {
+  Rhum.testSuite("constructor()", () => {
+    Rhum.testCase("sets module_name", () => {
+      Rhum.asserts.assertEquals(
+        b.getModuleName(),
+        "rhum",
+      );
+    });
+    Rhum.testCase("sets parsed_args", () => {
+      Rhum.asserts.assertEquals(
+        b.getParsedArgs(),
+        {},
+      );
+      Rhum.asserts.assertEquals(
+        c.getParsedArgs(),
+        {
+          branch: "v1.2.3",
+        },
+      );
+    });
+    Rhum.testCase("sets is_for_pre_release", () => {
+      Rhum.asserts.assertEquals(
+        b.isForPreRelease(),
+        false,
+      );
+      Rhum.asserts.assertEquals(
+        c.isForPreRelease(),
+        true,
+      );
+    });
+    Rhum.testCase("sets latest_versions", async () => {
+      const versions = {
+        rhum: await c.getModulesLatestVersion("rhum"),
+        deno: await c.getModulesLatestVersion("deno"),
+        deno_std: await c.getModulesLatestVersion("std"),
+      };
+      Rhum.asserts.assertEquals(
+        await c.getLatestVersions(),
+        versions,
+      );
+    });
+  });
+
+  await Rhum.testSuite("bump", async () => {
```

Comment:
awaits?

Reply:
yea there was an issue with the test not waiting for the fetch calls to finish. so the subsequent test suite would run while this one was still running.

Reply:
hmm... maybe the issue is with the last test suite and not this one. hldn. let me investigate.

Reply:
yea.. without these awaits, the tests run concurrently